Dockerfile-pytorch-ngc

```diff
@@ -0,0 +1,33 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# NGC images contain user owned files in /usr/lib
+RUN chown root:root /usr/lib
+
+# Copy various shell scripts that group dependencies for install
+COPY dockerfile_scripts /tmp/det_dockerfile_scripts
+
+RUN /tmp/det_dockerfile_scripts/install_deb_packages.sh
+RUN /tmp/det_dockerfile_scripts/add_det_nobody_user.sh
+RUN /tmp/det_dockerfile_scripts/install_libnss_determined.sh
+
+# We uninstall these packages after installing. This ensures that we can
+# successfully install these packages into containers running as non-root.
+# `pip` does not uninstall dependencies, so we still have all the dependencies
+# installed.
+RUN python -m pip install determined && python -m pip uninstall -y determined
+
+RUN python -m pip install -r /tmp/det_dockerfile_scripts/additional-requirements-torch.txt \
+    -r /tmp/det_dockerfile_scripts/additional-requirements.txt \
+    -r /tmp/det_dockerfile_scripts/notebook-requirements.txt
```

Comment:
totally optional nit, but if you're going to pass a bunch of similar flag/value pairs, it's nice for the reader if you separate the main command from the first arg as well (as you already have in `install_deb_packages.sh`):

```docker
RUN python -m pip install \
    -r /tmp/det_dockerfile_scripts/additional-requirements-torch.txt \
    -r /tmp/det_dockerfile_scripts/additional-requirements.txt \
    -r /tmp/det_dockerfile_scripts/notebook-requirements.txt
```